README.md

```diff
@@ -40,9 +47,52 @@ Here are some key features of YuniKorn.
 - Customized resource types (like GPU) scheduling support.
 - Rich placement constraints support.
 - Automatically map incoming container requests to queues by policies. 
-- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management. 
+- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management.
+
+### Integration with K8s:
+
+`yunikorn-k8shim` nicely integrated with K8s, supported features like: 
+
+- _Predicates:_ All kinds of predicates such as node-selector, pod affinity/anti-affinity, taint/tolerant, etc.
+- _Persistent volumes:_ We have verified hostpath, EBS, NFS, etc. 
+- _K8s namespace awareness:_ YuniKorn support hierarchical of queues, does it mean you need to give up K8s namespace? Answer is no, by default YuniKorn fully support automatically
+ map K8s
+ namespaces to YuniKorn queues. All Namespace related ResourceQuota, permissions will be still valid.
```

Comment:
This requires setting up the queue mappings, I would not say "automatically".
I would use with a simple config addition.

Reply:
I will make it to more accurate.